libs/movie/src/lib/movie/dashboard/actions/actions.component.ts

```diff
@@ -67,21 +67,7 @@ export class DashboardActionsShellComponent {
   }
 
   async updateStatus(status: StoreStatus, message?: string) {
-    await this.movieService.update(this.movie.id, (movie) => ({
-      ...movie,
-      app: {
-        ...movie.app,
-        [this.app]: {
-          ...movie.app[this.app],
-          status: status,
-        },
-      },
-    }));
-
-    if (message) {
-      this.snackbar.open(message, '', { duration: 4000 });
-    } else {
-      this.snackbar.open(`Title ${storeStatus[status]}.`, '', { duration: 4000 });
-    }
+    await this.movieService.updateStatus(this.movie.id, status);
+    this.snackbar.open(message ? message : `Title ${storeStatus[status]}.`, '', { duration: 4000 });
```

Comment:
This should also work: 
```suggestion
    this.snackbar.open(message || `Title ${storeStatus[status]}.`, '', { duration: 4000 });
```